docs/Workshop_Areas/Lab/biolab-usage-guidelines.md

```diff
@@ -0,0 +1,378 @@
+>
+> (a) organisms generated using methods based on natural mating or
+>     recombination;
+> (b) somatic cell nuclear transfer ('cloning') provided no GM material
+>     is present and the donor/recipient organisms are able to
+>     interbreed;
+> (c) artificial transfer of pollen from one flower to another
+>     (considered to be natural fertilisation);
+> (d) hybrid or reassortant viruses generated by natural recombination
+>     or transencapsidation during co-infection of a cell;
+> (e) DNA vaccination, where naked or synthetic DNA is introduced into
+>     animals to elicit an immune response against antigens encoded by
+>     that material, with no intention of stable integration.
+
+*Paragraph 22, The Genetically Modified Organisms (Contained Use)
+Regulations 2014 Guidance on Regulations*
+
+If the user is unsure of whether a proposed project requires CL1, the
+BSO should be sought before any work on the project commences.
+
+A user must have an approved risk assessment for any work involving
+genetic modification before commencing this work. More information on
+the process of risk assessment and approval can be found in SOP
+[LBL06001](lbl06001.md). A separate risk assessment must be performed
+and approved for transporting any GMO outside the laboratory (details in
+SOP [LBL06002](lbl06002.md)).
+
+CL1 activities must follow the procedures set out in
+[LBL05001](lbl05001.md) and the further procedures set out in the
+approved risk assessment.
+
+In general this includes:
+
+-   following GMP as set out in the [Working with Micro-organisms section](#working-with-micro-organisms) (above) when working with
+    micro-organisms,
+-   all biomaterial and waste that was in contact with a genetically
+    modified micro-organism (GMM) must be deactivated by autoclaving
+    before disposal (the autoclave must have passed the deactivation
+    test that day),
+-   taking care to prevent the dispersal of the GMO outside the
+    laboratory.
+
+## Refrigerator, Freezer and Sample Storage
+
+The refrigerator maintains samples and reagents at 4°C. Typically the
+refrigerator stores bacterial plates and reagents for transformation.
+Due to the presence of biomaterial in the refrigerator, caution should
+be exercised to prevent contamination of the user. The freezer contains
+longer term bacterial stocks and enzymatic reagents, maintaining a
+temperature of at least -20°C. Similar caution should be taken when
+using the freezer to avoid personal contamination. Where specified by
+the reagent packaging or MSDS of a reagent, reagents may also be stored
+in the freezer and refrigerator. Specific SOPs also cover the storage
+and recovery of frozen bacterial stocks and procedures requiring the
+usage of frozen and refrigerated reagents.
+
+we currently dont have a -20 freezer however one is available in the space and would like to use it for the lab and its storage outside the lab in a out the way area. 
```

Comment:
If it's currently available, it may be communal, metal work occasionally requires cooling parts for press fittings to tight tolerances, +2 thou, -2 thou, for example. It should be determined if this is a communal item, or not used elsewhere.